src/Enhavo/Bundle/SidebarBundle/Resources/config/app/config.yml

```diff
@@ -0,0 +1,23 @@
+doctrine:
+    orm:
+        resolve_target_entities:
+            Enhavo\Bundle\SliderBundle\Model\SliderInterface: Enhavo\Bundle\SidebarBundle\Entity\Sidebar
```

Comment:
`Enhavo\Bundle\SliderBundle\Model\SliderInterface` is not part of the sidebar bundle. Please add a interface `Enhavo\Bundle\SidebarBundle\Model\SidebarInterface` and the `Enhavo\Bundle\SidebarBundle\Entity\Sidebar` should implement this interface